.github/workflows/build-revive-wasm.yml

```diff
@@ -87,6 +87,7 @@ jobs:
           path: |
             ${{ env.REVIVE_WASM_INSTALL_DIR }}/resolc.js
```

Comment:
Do we still need to release it? Does it have any worth over `resolc_packed.js`? How would you use it?

Reply:
resolc.js makes sense for use with Node.js, where it is not dynamically downloaded, for example in `js-revive`